lib/version_utils.pm

```diff
@@ -200,9 +201,9 @@ sub check_version {
 }
 
 =head2 is_microos
-Check if distribution is SUSE MicroOS or openSUSE MicroOS with optional filter:
+Check if distribution is openSUSE MicroOS with optional filter:
```

Comment:
Could you verify the output here  http://podwebview.odyniec.net/ please?

Reply:
done